tutorials/commands/README.md

```diff
@@ -50,7 +50,7 @@ type model struct {
 `Cmd`s are functions that perform some I/O and then return a `Msg`. Checking the
 time, ticking a timer, reading from the disk, and network stuff are all I/O and
 should be run through commands. That might sound harsh, but it will keep your
-Bubble Tea program staightforward and simple.
+The Bubble Tea program is straightforward and simple.
```

Comment:
That sounds wrong now.

Reply:
Would 'Bubble Tea is straightforward and simple.` sound better or 'Bubble Tea program is straightforward and simple.'
I just thought adding 'The' is grammatically correct. 

I'm open to anything :)

Reply:
I think the original version was fine:

"That might sound harsh, but it will keep your Bubble Tea program staightforward and simple."

Reply:
Now that I re read the whole sentence, I agree with you. Thats an oversight on my part. I'll make a fix. Thanks!

Reply:
Would the 'is' before 'straightforward' not be needed anymore?

Reply:
Yep, the "is" needs be gone, too :smile: